src/AuthorsView.elm

```diff
@@ -57,31 +59,48 @@ renderAuthor model ( author, index ) =
     let
         addAffiliationButton =
             if model.affiliationLimit > (List.length author.affiliations) then
-                div [ class "add-affiliation-to-author button button--tertiary" ]
-                    [ div [ onClick (AddAffiliation author.id) ]
-                        [ text "Add Another Affiliation to Author" ]
+                button
+                    [ class "add-affiliation-to-author button button--secondary"
+                    , onClick (AddAffiliation author.id)
                     ]
+                    [ text "+ Add Affiliation" ]
             else
                 div []
                     []
+
+        chunkifiedAuthorFields =
+            chunk 3 model.authorFields
     in
-        div [ class "author form__question-sub-section" ]
-            [ div [ class "form__label" ] [ text ("Author " ++ toString index) ]
-            , div [ class "form__question-sub-section--inline" ]
-                --for each of the authorFields we want to add a div like this
-                [ div [ class "form__question-sub-section--inline" ] (List.map (renderFieldResponses model author.authorFieldResponses author.id) model.authorFields)
+        div [ class "author aa" ]
+            [ div [ class "aa__dividing-title" ]
+                [ span [ class "aa__subtitle" ]
+                    [ text ("Author " ++ toString index) ]
+                ]
+            , button
+                [ class "remove aa__remove-button aa__remove-button--top-indent button button--secondary button--delete"
+                , onClick (DeleteAuthor author.id)
+                ]
+                [ text ("Remove Author " ++ toString author.id) ]
+            , div [ class "aa__sub-section aa__sub-section--table" ]
+                --for each of the authorFields we want to add a div like this:
+                [ div [ class "aa__field aa__field--tablecell" ] (List.map (renderFieldResponsesLine model author.authorFieldResponses author.id) chunkifiedAuthorFields)
                 ]
-            , span [ class "remove button button--secondary" ]
-                [ div
-                    [ onClick (DeleteAuthor author.id) ]
-                    [ text "Remove Author" ]
+            , div [ class "aa__inner-container" ]
+                [ div [ class "aa__dividing-title aa__dividing-title--linebreak" ]
+                    [ span [ class "aa__subtitle" ]
+                        [ text ("Author " ++ toString author.id ++ " Affiliations") ]
+                    ]
+                , (renderAffiliations model author.affiliations author.id)
+                , addAffiliationButton
                 ]
-            , div [ class "" ]
-                [ (renderAffiliations model author.affiliations author.id) ]
-            , addAffiliationButton
             ]
 
 
+renderFieldResponsesLine model authorFieldResponses authorId authorFieldLine =
```

Comment:
I know I've missed this sometimes and not something that needs to be changed in this PR but we should try to add annotations to top level functions